### DISCUSSION TITLE
How to use the `foreach` loop?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#3019] [General]
I can't find the needed information on the help find. Or perhaps I really get lost in the Russian texts and my machine translator. Btw, it seems you are expecting people to already have background in Delphi since I can't find anything about basic stuffs like how to use conditional statements and loops. This is so wrong. Not anyone come to PascalABC.NET is a Delphi expert.

**Comment by SunSerega** [comment]
I don't have a background in Delphi. I mean I never even touched it.
I was expecting you to somehow already know that you should search for .Net in general or C# specifically if you cannot find something in the Pascal help - that's my bad.
In particular, you should search for .Net things (like, everything from `System`) on MSDN (or better using Google and selecting MSDN results) - that's the official help site from Microsoft.
